test/ffmpeg-qsv/encode/10bit/vp9.py

```diff
@@ -35,9 +35,16 @@ def before(self):
 
 class cqp_lp(VP9_10EncoderLPTest):
   def init(self, tspec, case, ipmode, qp, quality, slices, refmode, looplvl, loopshp):
-    slash.logger.notice("NOTICE: 'refmode' parameter unused (not supported by plugin)")
-    slash.logger.notice("NOTICE: 'loopshp' parameter unused (not supported by plugin)")
-    slash.logger.notice("NOTICE: 'looplvl' parameter unused (not supported by plugin)")
+    skip_test_by_parameter("encode/10bit/vp9/cqp_lp", ['refmode', 'looplvl', 'loopshp'],
```

Comment:
The key needs to be unique per "implemented" test function because each implementation may support different parameters.  The key used here is not unique to one specific test implementation and will "leak" into the other implementations... this is not good.

This issue is solved in the #298 decorator which uses the "func" pointer as the unique key.